R/add_co2-jitter_range_by_benchmark.R

```diff
@@ -5,3 +5,10 @@ jitter_range_by_benchmark <- function(data, ...) {
     map(jitter_range, ...) |>
     reduce(bind_rows)
 }
+
+toy_jitter_range_range_by_benchmark <- example_data_factory(tibble(
+  !!col_benchmark() := "all",
+  !!col_risk_category_emissions() := "low",
+  !!col_min() := 1L,
+  !!col_max() := 2L
+))
```

Comment:
If I need test-data for `f()`, then I define `toy_f()` in R/f.R. This aims to keep together code that changes for the same reasons. This seems more maintainable than dumping all `toy_*()` into R/utilt-tests.R.